doc/classes/Navigation2D.xml

```diff
@@ -36,6 +39,7 @@
 			<argument index="2" name="optimize" type="bool" default="true">
 			</argument>
 			<description>
+				Calculates a path between two local-coordinate points and returns the result as a [code]PoolVector2Array[/code]. If the [code]optimize[/code] is [code]true[/code] (the default), string pulling will be applied to smoothen the calculated path.
```

Comment:
Remove "-" between local and coordinate.
If you mention [PoolVector2Array] it should be a link, although it's already listed above as the return value. 
Remove "the" before optimize
"smoothen" isnt' a word, change to "smooth"

Suggestion for first sentence:
Returns the path between two given points. Points are in local coordinate space.